src/Resources/config/listener.yml

```diff
@@ -83,6 +83,7 @@ services:
             - "%contao.pretty_error_screens%"
             - "@twig"
             - "@contao.adapter.config"
+            - "@monolog.logger"
```

Comment:
You should use the generic `@logger` service (which usually is an alias to `monolog.logger`.